packages/react/src/components/Toggletip/index.tsx

```diff
@@ -173,7 +206,7 @@ Toggletip.propTypes = {
    * Provide a custom element or component to render the top-level node for the
    * component.
    */
-  as: PropTypes.oneOfType([PropTypes.string, PropTypes.elementType]),
+  as: PropTypes.elementType,
```

Comment:
will making this change still allow for strings such as "div" to be passed in?

Reply:
Yes! I tested passing a string into this component in a story and no warnings were printed about it. Any `keyof JSX.InstrinsicElements` should be a valid element. Any other string would not be a valid element.